__tests__/app.test.js

```diff
@@ -168,4 +168,44 @@ describe("/api/articles/:article_id/comments", () => {
         expect(response.body.msg).toBe("Bad request");
       });
   });
+  test("POST 201: inserts a new comment into the db and responds with the newly created comment", () => {
```

Comment:
I can't see any addition to the endpoints.json file so please be sure to update that with this endpoint 🕵️